src/pages/experiments/[experimentId]/plots-and-tables/embedding-categorical/index.jsx

```diff
@@ -107,28 +108,24 @@ const EmbeddingCategoricalIndex = ({ experimentId }) => {
       }],
     },
   ];
-  if (!config) {
-    return (
-      <center>
-        <Loader experimentId={experimentId} />
-      </center>
-    );
-  }
 
   const renderExtraPanels = () => (
     <>
       <Panel header='Group by' key='1'>
         <p>
           Select the cell set category you would like to group cells by.
         </p>
-        <Select
-          labelInValue
-          style={{ width: '100%' }}
-          placeholder='Select cell set...'
-          value={{ key: config.selectedCellSet }}
-          options={generateCellSetOptions()}
-          onChange={onCellSetSelect}
-        />
+        {config ? (
+          <Select
+            labelInValue
+            style={{ width: '100%' }}
+            placeholder='Select cell set...'
+            loading={config}
+            value={{ key: config.selectedCellSet }}
+            options={generateCellSetOptions()}
+            onChange={onCellSetSelect}
+          />
+        ) : <Skeleton.Input style={{ width: '100%' }} active />}
```

Comment:
This component would take a very long time to render while `config` was being loaded. Instead of delaying the whole page until `config` is loaded so this component does not crash, I added a custom renderer for the field that used `config` directly so it can load on its own.